core/domain/exp_fetchers.py

```diff
@@ -273,10 +274,29 @@ def get_exploration_summaries_matching_ids(exp_ids):
         corresponding to the given exploration ids. If an ExplorationSummary
         does not exist, the corresponding returned list element is None.
     """
-    return [get_exploration_summary_from_model(model)if model else None
+    return [get_exploration_summary_from_model(model) if model else None
             for model in exp_models.ExpSummaryModel.get_multi(exp_ids)]
 
 
+def get_exploration_summaries_subscribed_to(user_id):
+    """Returns a list of ExplorationSummary domain objects that the user
+    subscribes to.
+
+    Args:
+        user_id: str. The id of the user.
+
+    Returns:
+        list(ExplorationSummary). List of ExplorationSummary domain objects that
+        the user subscribes to.
+    """
+    return [
+        summary for summary in
+        get_exploration_summaries_matching_ids(
+            subscription_services.get_exploration_ids_subscribed_to(user_id)
```

Comment:
Since this exists in `subscription_services`, should this function also exist in `subscription_services`?

Reply:
It caused a circular dependency when implemented in `subscription_services`.

Reply:
Oh, ok, fine then.